bin/changelog.js

```diff
@@ -76,7 +76,22 @@ const writeEntry = async ( content_url ) => {
 					const labels = getLabels( data.labels );
 					const labelTag = labels.length ? `(${ labels })` : '';
 					const authorTag = collaborator ? '' : `👏 @${ data.user.login }`;
-					const entry = `- ${ type }: ${ data.title } #${ data.number } ${ labelTag } ${ authorTag }`;
+					let title;
+					if ( /### Changelog Note/.test( data.body ) ) {
+						const bodyParts = data.body.split( '### Changelog Note:' );
+						const note = bodyParts[ bodyParts.length - 1 ];
+						title = note
```

Comment:
We could, but since this output is subject to manual review I'd rather not introduce more complexity. It would throw out a supplied title for a typo like `Tweak :` or `Tweak - `.

Reply:
I think that's fair.  What happens if there is a typo?  I guess just manual review and fix by the person putting together the changelog?

Reply:
Yeah that would be it. The point of the automation is so the person creating the release doesn't have to spend 2 hours going through 100 issues/pulls. A quick read over should be required anyways. I can update the fieldguide to mention what to look for